src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/MethodDefHandler.java

```diff
@@ -110,26 +110,15 @@ public void checkIndentation() {
         checkModifiers();
         checkThrows();
 
-        if (getMethodDefParamRightParen(getMainAst()) != null) {
-            checkWrappingIndentation(getMainAst(), getMethodDefParamRightParen(getMainAst()));
-        }
+        final DetailAST nextToLastChild = getMainAst().getLastChild().getPreviousSibling();
+        checkWrappingIndentation(getMainAst(), nextToLastChild);
+
```

Comment:
We really only needed to get the next to last child in any case; whether it was the `RPAREN` or some child of an `IMPLEMENTS_CLAUSE`.